README.md

```diff
@@ -109,75 +109,6 @@ Version 1.0.0 of this module introduces a breaking change: adding the `disable-l
 In either case, upgrading to module version `v1.0.0` will trigger a recreation of all node pools in the cluster.
 
 [^]: (autogen_docs_start)
-
-## Inputs
```

Comment:
Please run `make generate_docs` to regenerate the tables after `make generate` regenerates the modules.

Reply:
Done